cypress/README.md

```diff
@@ -24,21 +24,26 @@ Use scenario tags:
 3. `@ignore` for temporary ignored scenarios
 4. `@[bug-number]` for example `@FE-1234` - use this tag after `@ignore` to explain why test is ignored.
 5. `@ignore` and `@[bug-number]` should be removed after bug fix
+6. `@build` for pipeline - each build is veirfied by this tests
```

Comment:
for a pipeline - each build will be verified by those tests :)

Reply:
today will be, but tomorrow... who knows